zebrad/Cargo.toml

```diff
@@ -1,13 +1,20 @@
 [package]
+# Crate metadata
 name = "zebrad"
 authors = ["Zcash Foundation <zebra@zfnd.org>"]
 license = "MIT OR Apache-2.0"
-version = "1.0.0-rc.2"
+version = "1.0.0-rc.3"
+repository = "https://github.com/ZcashFoundation/zebra"
+
+# Settings that impact compilation
 edition = "2021"
+
 # Zebra is only supported on the latest stable Rust version. Some earlier versions might work.
-# Zebra uses features introduced in Rust 1.58.
-rust-version = "1.58"
-repository = "https://github.com/ZcashFoundation/zebra"
+# Zebra currently uses features introduced in Rust 1.65 or later.
```

Comment:
Nitpick:
```suggestion
# Zebra currently uses features included in Rust 1.65 or later.
```

Reply:
Hmm, I'm not sure that's what I want to say here.

I want to say something like "we use a Rust feature that was introduced in 1.65, so Zebra won't compile with earlier versions. We might also use later features as soon as they are in stable, so this isn't a minimum supported Rust version, it's just a guideline."

Reply:
> I want to say something like "we use a Rust feature that was introduced in 1.65, so Zebra won't compile with earlier versions. We might also use later features as soon as they are in stable, so this isn't a minimum supported Rust version, it's just a guideline."

That sounds better, maybe "Zebra currently uses features introduced in Rust 1.65, and may later use features from the latest stable release of Rust."?

Reply:
Sounds good, I'll apply that change throughout this PR.

Reply:
What do you think of 66af9819b9?

I tried to make it a bit shorter.

Reply:
I think that's good now without the "currently", in principle Zebra uses features introduced in any later stable releases. 👍